types/index.d.ts

```diff
@@ -222,7 +222,7 @@ export interface BaseQueryOptions<TResult = unknown, TError = Error> {
    * Set this to `false` to disable automatic refetching when the query mounts or changes query keys.
    * To refetch the query, use the `refetch` method returned from the `useQuery` instance.
    */
-  enabled?: boolean
+  enabled?: any
```

Comment:
I would make it `boolean | unknown` instead

Reply:
done. I also add some explanation in the readme. Feel free to change the wording if necessary.